libgadget/blackhole.c

```diff
@@ -32,6 +32,17 @@ struct BlackholeParams
     double SeedBlackHoleMass;	/*!< Seed black hole mass */
     double BlackHoleEddingtonFactor;	/*! Factor above Eddington */
     int BlackHoleRepositionEnabled; /* If true, enable repositioning the BH to the potential minimum*/
+    
+    /**********************************************************************/
+    int MergeGravBound; /*if 1, apply gravitational bound criteria for BH mergers */
+    
+    int BH_DynFrictionMethod;/*0 for off; 1 for Star Only; 2 for DM+Star; 3 for DM+Star+Gas */
+    int BH_DFBoostFactor; /*Optional boost factor for DF*/
+    double BH_DFbmax; /* keep it as a parameter for tuning purpose */
```

Comment:
What are the units for this one? Is it kpc/h?

Reply:
Yes, this is in kpc/h.

Reply:
Please add a comment that it is in internal units

Reply:
Sorry, actually in Gizmo's implementation, this DFbmax is already in physical unit of kpc. (Already corrected it.)  However the value of this factor doesn't have large effect.